frontend.yml

```diff
@@ -0,0 +1,48 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: malware
```

Comment:
should this be `malware-detection-frontend`?  I believe this should match [here](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/insights/malware-detection/app.yml#L55).

Reply:
I'll change this as well, I think I was matching against cloud-services-config